firefox_accounts/views/fxa_multi_service_dau.view.lkml

```diff
@@ -0,0 +1,58 @@
+view: fxa_multi_service_dau {
+
+  derived_table: {
+
+    sql:
+    SELECT
+      DATE(timestamp) as submission_date,
+      user_id,
+      ARRAY_AGG(DISTINCT service IGNORE NULLS ORDER BY service) as services,
+      MAX_BY(country_code, timestamp) as country_code
+    FROM
+      `mozdata.firefox_accounts.fxa_all_events`
+    WHERE service IN ('sync', 'mdn-plus', 'guardian-vpn', 'fx-monitor', 'fx-private-relay', 'pocket-web', 'amo-web', 'thunderbird-addons', 'mozilla-iam', 'moz-social', 'mozilla-support', 'pontoon', 'mozilla-hubs-dev')
```

Comment:
This would be more readable and maintainable if the services were listed in alphabetical order, or even better on separate lines in alphabetical order.